rfcs/quick-start-book.md

```diff
@@ -0,0 +1,352 @@
+
+## Implementation strategy
+
+1. The user-facing explanation above should live in `CONTRIBUTING.md` of the [`bevy-website` repo](https://github.com/bevyengine/bevy-website).
+2. This should be linked to from the website, likely under a "Contributing" tab.
+3. Per [bevy-website #143](https://github.com/bevyengine/bevy-website/issues/143),the website will need to be revamped somewhat.
+4. Automatic compilation of code snippets is the critical feature from the above, ensuring that we can keep the examples current in a reliable way.
+5. Once this RFC is merged, work should begin on a separate branch of `bevy-website` repo to create a new book.
+6. For the launch of 0.6 (or perhaps sooner), this branch is merged and becomes the new `main`.
+7. A persistent `bevy-main` branch of the repo is maintained and with a dependency on the `main` branch of `bevy` to ensure that we can test out new changes and refine the code before new features and breaking changes go live and reduce release crunch.
+
+### Quick Start examples
+
+### Breakout
+
+**Genres:** arcade
+**Focus:** basic physics, keyboard controls, sprites
+
+Control a paddle with your keyboard to smash bricks and score points in this classic arcade game!
+
+### Falling Sand
+
+**Genres:** simulation, RTS, factory-builder
+**Focus:** advanced ECS patterns, pixel graphics, handling mouse input
+
+Simulate the interactions between different materials as you build your own creation.
+
+### 3D Puzzle Game
+
+**Genres:** puzzle, 3rd person platformer
+**Focus:** camera controls, rule checking, level creation, 3D graphics
+
+Create devious puzzles for your players to solve as they navigate a 3D world.
+
+### Bevy Book chapters
+
+Advanced chapters are skippable, rely on prior knowledge and are not useful for beginners.
+
+1. Getting Started
+   1. Installing Rust and Bevy
+   2. `App`, `AppBuilder` and `World`
+   3. Advanced: Fast compiles
+2. Entities, Components and Systems
+   1. Entities have components
+   2. Systems access data through queries
+   3. Resources are global singletons
+   4. Bundles collect related components
+   5. Spawning, despawning and modifying entities with `Commands`
+   6. Reliable change detection
+   7. Advanced: generic systems
+   8. Advanced: exclusive `World` access
+   9. Advanced: `bevy_ecs` as a standalone crate
+3. Game logic
+   1. Events
+   2. Time and timers
+   3. Run criteria and fixed timesteps
+   4. States
+   5. Advanced: stages and system ordering
+   6. Advanced: async tasks
+   7. Advanced: custom runners and headless operation
+4. Graphics
+   1. The `Transform` component controls position
+   2. 2D sprites
+   3. 3D meshes
+   4. Cameras
+   5. `Parent`-`Child` hierarchy
+   6. Configuring window(s)
+   7. Advanced: writing shaders
+5. Assets
+   1. Loading assets
+   2. Working with handles
+   3. Advanced: custom asset types
+   4. Advanced: hot reloading
+   5. Advanced: scenes and reflection
```

Comment:
Scenes will need their own section eventually, but I'm cool with leaving it as a footnote for now we wait for them to mature.